### PR TITLE
Fixes to Get started: optimize your React app

### PR DESCRIPTION
**Pull request by kyarik**
Changes proposed in this pull request:

- Added a missing comma.
- Added a missing closing parenthesis.
- Added a missing "the" before "actual web page".

### CODE REVIEW

**Comment by mfriesenhahn** [comment]
Hi, @kyarik! Thanks for the assist!

I think the closing parenthesis you suggested might be in the wrong spot. (The sentence is admittedly a little hard to parse because there's a pronoun-antecedent error.) To fix things I might do something like this:

> It simplifies quite a few processes involved with managing packages—including running CLI commands like `create-react-app`—without having to install them globally on your machine.

What do you think?

**Comment by kyarik** [comment]
@mfriesenhahn - You're right! The parentheses should have only enclosed the "like `create-react-app`" part.

Your improved version of the sentence looks good to me. I'll edit it. Thanks.

**Comment by mfriesenhahn** [comment]
Huzzah! Thanks again!
